week-1/Homework/mandatory/3-function-output.js

```diff
@@ -1,31 +1,43 @@
 // Add comments to explain what this function does. You're meant to use Google!
+//The Math.random() function returns a floating-point, pseudo-random number in the range 0 to less than 1
```

Comment:
This is a great description of `Math.random()` - can you also explain what the function `getNumber()` does? (It's very similar!) In particular, what's the lowest and highest value `getNumber()` may return?